hooks/src/lib/CardCVC/useCardCVCState/useCardCVCState.ts

```diff
@@ -0,0 +1,24 @@
+import { useState } from "react";
+
+interface useCardCVCStateProps {
+  validateCardCVC: (cardCVC: string) => boolean;
+}
+
+const useCardCVCState = ({ validateCardCVC }: useCardCVCStateProps) => {
+  const [cardCVC, setCardCVC] = useState("");
+
+  const handleCardCVC = ({
+    event,
+  }: {
+    event: React.ChangeEvent<HTMLInputElement>;
+  }) => {
+    const { value } = event.target;
```

Comment:
아래처럼 작성하지 않으신 이유가 있을까요? 

```suggestion
    event: { target: { value }}: React.ChangeEvent<HTMLInputElement>;
  }) => {
```

Reply:
오! 이 방식은 처음 써보는데, 매개변수에서 미리 값을 구조 분해하니까
`const { value } = event.target` 코드를 따로 작성하지 않아도 돼서 깔끔하네요 👍

Reply:
67198591f024c85e5f3e8d71d37b1d6fda26a12a